app/controllers/schools/participants_controller.rb

```diff
@@ -18,8 +19,6 @@ def show
     @mentor = @participant.early_career_teacher_profile&.mentor
   end
 
-  def edit_details; end
-
```

Comment:
Is this unused?

Reply:
It was used to display "Not implemented" page